docs/source/tutorials.rst

```diff
@@ -282,7 +282,7 @@ Now that we have all the objects at our disposal, we can create the DrawBlendsGe
   draw_generator = btk.draw_blends.CatsimGenerator(
       catalog,
       sampling_function,
-      [Rubin],
+      [LSST],
```

Comment:
different from line 451 below, might want to homogenize

Reply:
related to #280